packages/devextreme/js/ui/text_box/ui.text_editor.mask.js

```diff
@@ -525,9 +526,15 @@ const TextEditorMask = TextEditorBase.inherit({
     },
 
     _updateMaskOption: function() {
+        const valueChangeEvent = addNamespace(this.option(this._getValueChangeEventOptionName()), `${this.NAME}ValueChange`);
+        const $input = this._input();
+
         this._updateHiddenElement();
         this._renderMask();
         this._validateMask();
+        // reattach events to get correct handlers order (T1214019)
```

Comment:
We usually use a `NOTE:` prefix for informative comments.
 But actually I don't see a necessity to add a comment here because this solution does not look like something specific and non-intuitive. Moreover we add a special test for this